plugins/ssl/certificate_file_expiry

```diff
@@ -72,23 +87,70 @@ if [ "$1" = "config" ] ; then
   echo "graph_category security"
 fi
 
+if [ "${warning:-}" = "" ] ; then
+  # as default when certificate is only valid for 5 days or less emit a warning
+  warning="5:"
+fi
+if [ "${critical:-}" = "" ] ; then
+  # as default when certificate is only valid for 1 day or less emit a critical
+  critical="1:"
+fi
+
 now=$(date +%s)
+get_validity() {
+  openssl_type=$1
+  file=$2
+  if [ "$file" != "--" ] ; then
```

Comment:
A single dash is a more common convention for specifying "read from stdin". Maybe this would be easier to read?